apps/web/app/s/[surveyId]/components/MediaBackground.tsx

```diff
@@ -60,6 +60,11 @@ export const MediaBackground: React.FC<MediaBackgroundProps> = ({
       const img = new Image();
       img.onload = () => setBackgroundLoaded(true);
       img.src = background?.bg;
+    } else if (background?.bgType === "upload" && background?.bg) {
+      // For upload, we create a new Image object to listen for the 'load' event
+      const img = new Image();
+      img.onload = () => setBackgroundLoaded(true);
+      img.src = background?.bg;
```

Comment:
Since this is exactly same as the above if else block, is there a reason why we cannot just have the condition check as an **or** `||` block?